src/bento/siteWideConfig.js

```diff
@@ -5,4 +5,17 @@ export default {
   enableAuthentication: true,
   // List for options for authentication empty array defaults to google
   authEndPoint: ['google'], // authEndPoint: []
```

Comment:
@ogwuazors Can we remove "authEndPoint" property from here. It's no longer used.